__tests__/actionCreators/resources.test.js

```diff
@@ -176,10 +181,11 @@ describe('newResource', () => {
 
       await store.dispatch(newResource(resourceTemplateId))
       const actions = store.getActions()
-      expect(actions.length).toEqual(2)
+      expect(actions.length).toEqual(3)
```

Comment:
Would it be clearer to write expect(actions).toEqual([{type: 'SET_RESOURCE'}, ...])

Reply:
Here, yes.